docs/source/api.rst

```diff
@@ -1,32 +1,70 @@
 API Reference
-=============
+#############
```

Comment:
Let's better be consistent with the rest and keep the `===` headers for first level sections.